Readme.md

```diff
@@ -50,11 +46,11 @@ During the first build of your project after adding `Lombiq.NodeJs.Extensions`,
 
 You may encounter the following error:
 
-```
-ΓÇëERRORΓÇë ENOENT: no such file or directory, realpath [...]
+```text
+ERROR ENOENT: no such file or directory, realpath [...]
```

Comment:
Did the linter fix this, or you, Dávid? Because I left the garbled text in there so it will be easier to find.

Reply:
No, this was me. `ΓÇë` is a terminal formatting instruction. Depending on the terminal emulator and character encoding you are using, it may or may not appear.

Reply:
I see. Well, for all the Windows user out there, including me, searching for the whole error line might not bring up this file then - and I wanted to help those sad, frustrated developers. But let's hope they will choose to search for "ENOENT: no such file or directory, realpath" instead.

Reply:
We could always shorten it to just
```
ENOENT: no such file or directory, realpath [...]
```

Reply:
Sounds good.